src/OpenTelemetry/Trace/Export/BatchingSpanProcessor.cs

```diff
@@ -213,7 +213,7 @@ private async Task ExportBatchAsync(CancellationToken cancellationToken)
             }
             catch (Exception ex)
             {
-                OpenTelemetrySdkEventSource.Log.SpanProcessorException("OnStart", ex);
+                OpenTelemetrySdkEventSource.Log.SpanProcessorException("ExportBatchAsync", ex);
```

Comment:
Would this work so we don't get out of sync again:

```suggestion
                OpenTelemetrySdkEventSource.Log.SpanProcessorException(nameof(ExportBatchAsync), ex);
```